index.html

```diff
@@ -171,7 +171,7 @@ <h3>
          </div>
          <button type="submit" value="submit">Get in touch</button>
       </form>
```

Comment:
Please add a `method` property to the `form` which should have a value `POST`. THere is an error and the form is not submitted because you did not use the `POST` method.

Reply:
Good morning @SajjadAhmad14 
thanks for the information 
pls tell me what's the value of action should be 
it wasn't mentioned so I added my email to it